assemble/bin/accumulo-util

```diff
@@ -411,12 +411,16 @@ function create_config() {
 }
 
 function build_native() {
-  native_tarball="$basedir/lib/accumulo-native.tar.gz"
   final_native_target="$basedir/lib/native"
+  if ls $final_native_target/libaccumulo* &> /dev/null; then
```

Comment:
yes, it is a test operator does not with regex so `ls` must be used but I will change to your suggestion